src/backend/parser/cypher_clause.c

```diff
@@ -4874,18 +4908,20 @@ static Node *make_edge_expr(cypher_parsestate *cpstate, RangeTblEntry *rte)
                                         InvalidOid, COERCE_EXPLICIT_CALL);
     label_name_func_expr->location = -1;
 
-    props = scanRTEForColumn(pstate, rte, AG_EDGE_COLNAME_PROPERTIES, -1, 0,
-                             NULL);
+    props = scanNSItemForColumn(pstate, pnsi, 0, AG_EDGE_COLNAME_PROPERTIES, -1);
 
-    args = list_make5(id, start_id, end_id, label_name_func_expr, props);
+    args = list_make4(id, start_id, end_id, label_name_func_expr);
+    args = lappend(args, props);
 
     func_expr = makeFuncExpr(func_oid, AGTYPEOID, args, InvalidOid, InvalidOid,
                              COERCE_EXPLICIT_CALL);
     func_expr->location = -1;
 
     return (Node *)func_expr;
 }
-static Node *make_vertex_expr(cypher_parsestate *cpstate, RangeTblEntry *rte)
+static Node *make_vertex_expr(cypher_parsestate *cpstate,
+                              ParseNamespaceItem *pnsi,
+                              char *label)
```

Comment:
Why is label parameter added here. I can't see its use in the function. Is there any specific reason I am overseeing ?

Reply:
Was an oversight on my part, fixed.